### ISSUE TITLE
Error when trying to register

### ISSUE THREAD

**Issue by JakobJBauer** [Maker-Mark/Stock-Portfolio-App#6]
Hello @Maker-Mark ,

I just wanted to try out this App, but when I registered my account on [this heroku link](https://oscar-stock-app.herokuapp.com/) it claims, that the email-Adress is already in use.

This Error occurs regardless of which email I enter, so I guess it is an error with the database connection.

**Comment by Maker-Mark** [comment]
@JakobJBauer Yes I think I deleted the collection uri for the app. Feel free to fork it and set it up with your own mongo db, it should work fine!
